python/arcticdb/util/utils.py

```diff
@@ -0,0 +1,325 @@
+"""
+Copyright 2024 Man Group Operations Limited
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+from arcticdb import Arctic
```

Comment:
Remove

Reply:
Will remove - I thought tests also need this header .... Will not put it anymore